cmd/protoc-gen-micro/plugin/micro/micro.go

```diff
@@ -137,6 +137,11 @@ func (g *micro) generateService(file *generator.FileDescriptor, service *pb.Serv
 	}
 	servName := generator.CamelCase(origServName)
 	servAlias := servName + "Service"
+	fullServName := servName
+	useGrpc := g.gen.Param["use_grpc"]
```

Comment:
useGrpc := g.gen.Param["use_grpc"] != ""
Do you think it's better to define userGrpc as bool?